firmware/smbios/smbios_portable_battery.ksy

```diff
@@ -0,0 +1,71 @@
+meta:
+  id: smbios_portable_battery
+  endian: le
+  imports:
+    - strings
+seq:
+  - id: length
+    type: u1
+  - id: handle
+    type: u2
+  - id: location_index
+    type: u1
+  - id: manufacturer_index
+    type: u1
+  - id: manufacture_date_index
+    type: u1
+  - id: serial_number_index
+    type: u1
+  - id: device_name_index
+    type: u1
+  - id: device_chemistry
+    type: u1
+  - id: design_capacity
+    type: u2
```

Comment:
Some documentation on what units are used (i.e. kWh, something else?) would be nice.